sctp/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.6.1
+version: 1.0.0
```

Comment:
The others changed just in minor version (0.6 -> 0.7, and 1.0 -> 1.1) - is there a reason SCTP changes in major version (0.6 -> 1.0)?